packages/apidom-ls/src/config/openapi/parameter/lint/is-defined-within-path-template.ts

```diff
@@ -0,0 +1,23 @@
+import { DiagnosticSeverity } from 'vscode-languageserver-types';
+
+import ApilintCodes from '../../../codes';
+import { LinterMeta } from '../../../../apidom-language-types';
+import { OpenAPI2, OpenAPI3 } from '../../target-specs';
+
+const isDefinedWithinPathTemplate: LinterMeta = {
```

Comment:
```suggestion
const definedWithinPathTemplate: LinterMeta = {
```

Reply:
We're not using `is/are/has` prefixes with the predicate naming by convention. Example:

we use `required-fields` instead of `are-required-fields`.

Reply:
Name of the file `defined-in-path-template.ts`